app/domain/authentication/authn_jwt/vendor_configurations/configuration_jwt_generic_vendor.rb

```diff
@@ -4,60 +4,60 @@ module VendorConfigurations
       # Mock JWTConfiguration class to use it to develop other part in the jwt authenticator
       class ConfigurationJWTGenericVendor < ConfigurationInterface
 
-        def initialize
+        def initialize(authenticator_input)
           @authentication_parameters_class = Authentication::AuthnJwt::AuthenticationParameters
           @extract_token_from_credentials = Authentication::AuthnJwt::InputValidation::ExtractTokenFromCredentials.new
           @validate_and_decode_token_class = Authentication::AuthnJwt::ValidateAndDecode::ValidateAndDecodeToken
           @identity_provider_factory = Authentication::AuthnJwt::IdentityProviders::CreateIdentityProvider
           @validate_resource_restrictions_class = Authentication::ResourceRestrictions::ValidateResourceRestrictions
           @create_identity_provider_class = Authentication::AuthnJwt::IdentityProviders::CreateIdentityProvider
           @logger = Rails.logger
+          authentication_parameters(authenticator_input)
```

Comment:
i would change the method name to `init_authentication_parameters` as it doesn't return an object rather than initializing it

Reply:
Done